### PR TITLE
feat(cli): Enable ceramic --version flag

### PR DESCRIPTION
**Pull request by ukstv**

### CODE REVIEW

**Review by v-stickykeys — APPROVED** [review]
woohoo! 🚀

File: packages/cli/src/bin/ceramic.ts [hunk 1]
```diff
@@ -2,8 +2,10 @@ import { program } from 'commander'
 import pc from 'picocolors'
 
 import { CeramicCliUtils } from '../ceramic-cli-utils.js'
+import { version } from '../version.js'
 
 program
+  .version(version, '-v, -V, --version', 'output the version number')
```
Comment (stbrody) [inline packages/cli/src/bin/ceramic.ts:8]: don't we need code in this file to actually print the version when the flag is passed?

Comment (ukstv) [reply packages/cli/src/bin/ceramic.ts:8]: We don't. It is handled by commander for us.

**Review by stbrody — APPROVED** [review]
neat!

**Pull request merged**